packages/boot/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@loopback/boot",
+  "version": "4.0.0-alpha.1",
+  "description": "A collection of Booters for LoopBack 4 Applications",
+  "engines": {
+    "node": ">=6"
```

Comment:
Echoing Miroslav's comments from Kyu's PR. Best to drop support of node v6 in this new package?

Reply:
My understanding on @bajtos comment on Kyu's PR was that we support Node 6 in new packages to avoid CI failure till such time that we can create a new PR to drop Node 6 across all packages. (I'm going to do that soon ... because it's an annoyance to support it when we are looking to drop it).